Symfony/CS/Fixer/Symfony/PhpdocIndentFixer.php

```diff
@@ -40,8 +40,8 @@ public function fix(\SplFileInfo $file, $content)
 
             // ignore inline docblocks
             if (
+                $prevToken->equalsAny(array(';', '{')) ||
                 ($prevToken->isWhitespace(array('whitespaces' => " \t")) && !$tokens[$index - 2]->isGivenKind(T_OPEN_TAG))
-                || $prevToken->equalsAny(array(';', '{'))
```

Comment:
(method-call && !method-call) || (method-call) - worth case 3 calls, best case 2 calls
(method-call) || (method-call && !method-call) - worth case 3 calls, best case 1 call

But that's partially lie - distribution inline/general doc-block in real projects can break this assumption easily.